### ISSUE TITLE
"This app can't run on your PC". Latest Windows Insider Build

### ISSUE THREAD

**Issue by JMccormick264** [ShareX/ShareX#2934]
For some strange reason, the builds in AppVeyor I get this message https://i.imgur.com/9pb7wa0.png
[This Build](https://ci.appveyor.com/project/ShareX/sharex/build/job/7mr6bhypgvon5evc/artifacts) Is the one I am using. Not sure why this is happening.

Windows 10 Insider Preview
Version: 1709
OS Build: 17040.100

**Comment by sylveon** [comment]
Are you running a 32-bits version of Windows? IIRC these builds are 64-bits

**Comment by JMccormick264** [comment]
I have always been able to use the builds off of here

**Comment by Jaex** [comment]
I tested that build now and it could install and run fine.

**Comment by JMccormick264** [comment]
Let me try again

**Comment by JMccormick264** [comment]
I figured it out. Some reason while using the Nightly version of Firefox, something happens with the download. Using Microsoft Edge, all is good. Not sure what is up with Firefox at the moment.
